config/mainCfg_TauTau_Legacy2018_tauIDSF.cfg

```diff
@@ -45,7 +53,7 @@ data_obs  =  DTauB, DTauC, DTauD
 
 [pp_QCD]
 QCDname      = QCD
```

Comment:
cross check the regions used for the QCD

Reply:
In the other mainCfg files we generally use the inverted QCD, should I change it to be inverted here too?

Reply:
yep, even though it doesn't change much